src/routes/gifs.js

```diff
@@ -0,0 +1,52 @@
+const express = require('express')
+const postService = require('../services/posts')
+const { logger } = require('../lib')
+const isAuthenticated = require('../middleware/isAuthenticated')
+const validateSchema = require('../middleware/validateSchema')
+
+const {
+	createGifsSchema
+	
+} = require('../schema')
+
+const log = logger()
+const router = express.Router()
+
+const transformGifResponse = (gif) => ({
+	userId: gif.userId,
+	title: gif.title,
+	image: gif.content,
+	published: gif.published,
+	createdOn: gif.createdAt,
+	gifId: gif.id
+	
+})
+
+const createGif = async (req, res, next) => {
```

Comment:
lets use the `catchAsync` moving forward

Reply:
I am done with this as well, kindly review